tools/nodetool

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+SCRIPT_PATH="$(dirname $(realpath $0))"
+
+# We want this script to work OOTB both in-repo and when installed, while
+# allowing the user to override this behaviour.
+# So try to come up with meanigful default according to the detected location,
+# but allow for override via env variables.
+if [[ "$SCRIPT_PATH" == "/opt/scylladb/bin" ]]
+then
+    DEFAULT_SCYLLA_PATH="/opt/scylladb/bin/scylla"
+    DEFAULT_NODETOOL_PATH="/opt/scylladb/bin/nodetool"
+else
+    DEFAULT_SCYLLA_PATH="$(realpath ${SCRIPT_PATH}/../build/dev/scylla)"
+    DEFAULT_NODETOOL_PATH="${SCRIPT_PATH}/java/bin/nodetool"
+fi
```

Comment:
what about if installed via ./install.sh?

Reply:
Maybe you can just `test -f "$SCRIPT_PATH"/scylla` instead of checking specifically the path /opt/scyllabdb/bin - then if the script is installed alongside scylla itself, it will be found - no matter where.